engine/tests/ElggCoreGetEntitiesTest.php

```diff
@@ -348,6 +348,40 @@ public function testElggAPIGettersTSPValidTypeValidSubtype() {
 	}
 
 	/**
+	 * Valid type, ELGG_ENTITIES_NO_VALUE as subtype
+	 */
+	public function testElggAPIGettersTSPValidTypeNoValueSubtype() {
```

Comment:
Will do. I want to add another test with just an empty array.
On Aug 14, 2015 1:38 PM, "Juho Jaakkola" <notifications@github.com> wrote:

> In engine/tests/ElggCoreGetEntitiesTest.php
> <https://github.com/Elgg/Elgg/pull/8852#discussion_r37068774>:
>
> > @@ -348,6 +348,40 @@ public function testElggAPIGettersTSPValidTypeValidSubtype() {
> >  	}
> >
> >  	/**
> > +	 * Valid type, ELGG_ENTITIES_NO_VALUE as subtype
> > +	 */
> > +	public function testElggAPIGettersTSPValidTypeNoValueSubtype() {
>
> Add a docblock that describes the test. That makes it easier to improve
> the name later.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8852/files#r37068774>.
>